src/openklant/utils/validators.py

```diff
@@ -39,7 +39,15 @@ def validate_charfield_entry(value, allow_apostrophe=False):
 
 
 validate_phone_number = RegexValidator(
-    regex="(0[8-9]00[0-9]{4,7})|(0[1-9][0-9]{8})|(\\+[0-9]{9,20}|1400|140[0-9]{2,3})",
+    regex=r"^("
```

Comment:
I'm not 100% sure about all the different allowed options for phone numbers, but I'm going to approve since the incorrect phone numbers mentioned in the issue are now correctly validated, seems like the missing anchors `^` and `$` to match the full string were the culprit :+1: